content/wasm-functions/list-and-inspect.md

```diff
@@ -72,8 +72,10 @@ $ spin aka apps info --name hello-fermyon-wasm-functions
 <!-- @nocpy -->
 
 ```console
-Name: hello-fermyon-wasm-functions
+Name: hello-fermyon-wasm-functions (25a5fd1e-d476-40fd-bc54-6cee0e846540)
```

Comment:
No action for now, but I've raised an issue for ID to be a separate field like it is in the JSON